### PR TITLE
Show how to compute profile for an image along polyline, remove unnecessary utils.py

### PR DESCRIPTION
**Pull request by gena**
... now sure is `geometry/` is a proper name, maybe `image/` ?

### CODE REVIEW

File: examples/geometry/image_profile.ipynb [hunk 1]
```diff
@@ -0,0 +1,128 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 40,
```
Comment (tylere) [inline examples/geometry/image_profile.ipynb]: The cells appear to have been run out of order.
If you are saving a notebook file with outputs, be sure to reset the kernel, clear the outputs, and run all the cells before saving and committing the file.

File: examples/geometry/image_profile.ipynb [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "line = ee.Geometry.LineString(\n",
+    "        [[-114.3843, 36.4257],\n",
+    "         [-114.3616, 36.4220],\n",
+    "         [-114.3514, 36.4203],\n",
+    "         [-114.3419, 36.4199],\n",
+    "         [-114.3116, 36.4228]]);\n",
```
Comment (tylere) [inline examples/geometry/image_profile.ipynb]: Unnecessary semicolon.

File: examples/geometry/image_profile.ipynb [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 31,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def reduceImageProfile(image, line, reducer, scale):\n",
+    "    length = line.length()\n",
+    "    distances = ee.List.sequence(0, length, scale)\n",
+    "    lines = line.cutLines(distances).geometries();\n",
```
Comment (tylere) [inline examples/geometry/image_profile.ipynb]: Unnecessary semicolon.

File: examples/geometry/image_profile.ipynb [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x23f3ac88588>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
```
Comment (tylere) [inline examples/geometry/image_profile.ipynb:113]: The reduceImageProfile function seems to be the most important part of this example. This could be improved by adding notebook annotations that explain what the function is doing.

File: examples/export/utils.py [hunk 5]
```diff
@@ -1,31 +0,0 @@
-import urllib, io, os
```
Comment (tylere) [inline examples/export/utils.py:1]: This deletion appears unrelated to the PR. Could you either move this to a separate PR or expand the description of this PR?

File: examples/getting-started/display-static-map.ipynb [hunk 6]
```diff
@@ -74,7 +72,7 @@
     "# Generate a URL that displays a global DEM\n",
     "url = ee.Image(\"USGS/SRTMGL1_003\").getThumbUrl({'min':0, 'max':3000})\n",
     "# Display the image in the notebook.\n",
-    "Image(url=url)"
+    "Image(url=url, embed=True, format='png')"
```
Comment (tylere) [inline examples/getting-started/display-static-map.ipynb:75]: This file appears unrelated to the PR. Could you move this to a separate PR, or expand the description of this PR if you think it is related?

File: examples/export/export-image-gdrive.ipynb [hunk 7]
```diff
@@ -2,13 +2,13 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 7,
```
Comment (tylere) [inline examples/export/export-image-gdrive.ipynb:5]: The cells appear to have been run out of order.
If you are saving a notebook file with outputs, be sure to reset the kernel, clear the outputs, and run all the cells before saving and committing the file.

**Review by tylere — CHANGES_REQUESTED** [review]

File: examples/geometry/image_profile.ipynb (hunk 2 above)
Comment (gena) [reply examples/geometry/image_profile.ipynb]: Thanks, copy/pasting from JS, fixed!

File: examples/geometry/image_profile.ipynb (hunk 1 above)
Comment (gena) [reply examples/geometry/image_profile.ipynb]: Yes, this sounds better

File: examples/export/utils.py (hunk 5 above)
Comment (gena) [reply examples/export/utils.py:1]: extended descriptions, utils.py is not needed anymore because if embed=True

File: examples/geometry/image_profile.ipynb (hunk 3 above)
Comment (gena) [reply examples/geometry/image_profile.ipynb]: fixed

**Comment by gena** [comment]
I've reorganized notebook a bit:
* added a short description at the beginning
* added documentation for reduceImageProfile()
* moved the actual example to the end

File: examples/export/utils.py (hunk 5 above)
Comment (tylere) [reply examples/export/utils.py:1]: ok, makes sense.

**Review by tylere — APPROVED** [review]
